ci-operator/config/kcp-dev/code-generator/kcp-dev-code-generator-main.yaml

```diff
@@ -0,0 +1,21 @@
+build_root:
```

Comment:
I would suggest the following:

```yaml
binary_build_commands: make build
build_root:
  from_repository: true
  use_build_cache: true
```

This will save the git fetch and go build cache between merges to `main` which should help with timing. Then, in the repo, add a file like this: https://github.com/kcp-dev/kcp/blob/main/.ci-operator.yaml

This file now controls your base image, which means you can send PRs to the repo itself to see what e.g. updating Go versions does. This base image will work best for us as compared to the ones that OCP uses.

Reply:
Made the change, and created a pr to add this: https://github.com/kcp-dev/code-generator/pull/26/files. Just curious, (sorry, I'm not familiar with how ocp ci works) how does the ci know how to build base image? Do we need to specify the contents of dockerfile somewhere or is this it?